mllib/src/main/scala/org/apache/spark/ml/classification/MultilayerPerceptronClassifier.scala

```diff
@@ -273,29 +273,29 @@ object MultilayerPerceptronClassifier
  * Each layer has sigmoid activation function, output layer has softmax.
  *
  * @param uid uid
- * @param layers array of layer sizes including input and output layers
+ * @param modelLayers array of layer sizes including input and output layers
```

Comment:
I know the constructor is private, but is it necessary to change this name?

Reply:
I think it is needed, since all estimators and their models should share the same params.

Reply:
What about just removing `modelLayers` in model?
since the value (array of layer sizes) can be easily obtain by $(layers)